tests/phpunit/CRM/Export/BAO/ExportTest.php

```diff
@@ -2508,8 +2508,8 @@ protected function getBasicSqlColumnDefinition($isContactExport) {
       'addressee' => '`addressee` varchar(255)',
       'email_greeting' => '`email_greeting` varchar(255)',
       'postal_greeting' => '`postal_greeting` varchar(255)',
-      'current_employer' => '`current_employer` varchar(128)',
-      'location_type' => '`location_type` text',
+      'current_employer' => '`current_employer` varchar(255)',
```

Comment:
@demeritcowboy current_employer actually winds up longer by removing the special handling - I think that is fairly neutral

Reply:
I can poke around with this one but I don't really understand the export subsystem so it just be r-run.

Reply:
@demeritcowboy yep -that would be great. there are really only 2 things 
1) check the changes in the test to see that they seem OK - ie - changing current_employer from 128 to 255 is probably OK. Since every field is tested every change will be visible in the test changes
2) light r-run